src/locale/index.js

```diff
@@ -109,18 +140,30 @@ const RU = {
       to save and distribute the read... :)
     `,
   },
+  errorMessages: {
+    yourName: 'Please, enter a valid name RU',
+    email: 'Please, enter a valid email RU',
```

Comment:
cascade

Reply:
let's not use cascade in locale to have the same each language props

Reply:
What I need to do? :)